src/app/screens/History/components/NoteItem/index.js

```diff
@@ -88,7 +89,11 @@ const NoteItem = ({ note, loading }) => {
         setTextStyle={setTextStyle}
         textClassNames={{ italic, bold }}
         onSave={handleModification}
-        canSave={textValue?.length > 0 && titleValue?.length > 0}
+        canSave={
+          textValue?.length > 0 &&
+          titleValue?.length > 0 &&
+          hasDifferences(note, { content: textValue, title: titleValue, italic, bold })
+        }
```

Comment:
move the validation into a const.
`const validationToSave :  textValue?.length > 0 && titleValue?.length > 0 && hasDifferences(note, { content: textValue, title: titleValue, italic, bold })`

```suggestion
        canSave={ validationToSave }
```